### PR TITLE
Instantiate models from the frontend

### PR DESCRIPTION
**Pull request by NextFire**
## Tasks

- [x] Add an option to instantiate a model from the frontend
- [x] If a kernel is connected, reproduce it in the backend and sync it
- [x] JSON export/import from the frontend
- [x] Comment and update docs

### CODE REVIEW

File: src/builtins/index.ts [hunk 1]
```diff
@@ -5,29 +5,33 @@ import {
...
     storeManager: ITLabStoreManager,
-    ctrlManager: ITLabCtrlManager
+    ctrlManager: ITLabCtrlManager,
+    pyManager: ITLabPythonManager
   ) => {
-    // register builtins models
+    // register models in store manager and controller manager
     for (const model of Object.values(models)) {
       storeManager.registerModel(model);
+      ctrlManager.registerModel(model);
```
Comment (adriendelsalle) [inline src/builtins/index.ts:31]: why this double registration is necessary? the control mgr can't get the relevant info from the store mgr?

**Review by adriendelsalle — APPROVED** [review]
just a comment left, thanks!

File: src/builtins/index.ts (hunk 1 above)
Comment (NextFire) [reply src/builtins/index.ts]: Changed in https://github.com/twiinIT/tlab/pull/7/commits/64124456343488f42879894e54c23615bddfe146

**Comment by NextFire** [comment]
Argh, I forgot that I enabled auto-merge...

**Pull request merged**